cmake/utils.cmake

```diff
@@ -1,25 +1,25 @@
 
 # Check if a dependency exist before trying to init git submodules
 function(check_init_submodule path)
-	file(GLOB DIR_CONTENT path)
-	list(LENGTH RESULT CONTENT_COUNT)
-	if (CONTENT_COUNT EQUAL 0)
-		if (NOT EXISTS "${PROJECT_SOURCE_DIR}/.git")
-			message(FATAL_ERROR "Failed to find third party dependency in '${path}'")			
-		endif()
+    file(GLOB DIR_CONTENT "${path}/*")
+    list(LENGTH DIR_CONTENT CONTENT_COUNT)
+    if (CONTENT_COUNT EQUAL 0)
+        if (NOT EXISTS "${PROJECT_SOURCE_DIR}/.git")
+            message(FATAL_ERROR "Failed to find third party dependency in '${path}'")           
+        endif()
 
-		find_package(Git QUIET)
-		if (NOT Git_FOUND)
-			message(FATAL_ERROR "Failed to find Git, third party dependency could not be setup at `${path}")
-		endif()
+        find_package(Git QUIET)
+        if (NOT Git_FOUND)
+            message(FATAL_ERROR "Failed to find Git, third party dependency could not be setup at `${path}")
+        endif()
 
-		message(STATUS "Setting up dependencies as git submodules")
-        execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init --recursive
+        message(STATUS "Setting up dependencies as git submodules")
+        execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init
```

Comment:
What's with the whitespace changes?

![image](https://github.com/R2Northstar/NorthstarLauncher/assets/40122905/b0fb6544-4acd-4f85-af17-bf73db3c5dbe)

Reply:
the previous version had a mix of spaces and tabs
the other cmake files in this project use spaces exclusively, so I followed that.

Reply:
I'm assuming it's cause the remainig `CMakeLists.txt` all use spaces? Either way if we change it, it should be a separate PR IMO ^^

Reply:
Aight, all whitespace changes are resolved now :D